Editor/Data/Type/DefaultObservableListDrawer.cs

```diff
@@ -24,6 +25,7 @@ public override void OnGUI(Rect position, SerializedProperty property, GUIConten
         {
             label.tooltip = EditorHelper.GetTooltipAttribute(fieldInfo)?.tooltip ?? string.Empty;
             EditorGUILayout.PropertyField(property, label, true);
+            string fieldLabel = label.text;
```

Comment:
Can we delete this and just inline `label.text` in the single usage of this variable below?

Reply:
for some reason when the `Foldout` runs, the scope of `label` changes to the label of the Foldout (which is `Elements`)

so unless I store it prior to that, I lose access to `label.text`

Reply:
https://answers.unity.com/questions/1408104/propertydrawerongui-label-parameter-contents-chang.html something something heap allocation